tests/Components/Models/UserTest.php

```diff
@@ -37,6 +37,16 @@ public function it_can_return_the_amount_of_solutions_that_were_given()
         $this->assertEquals(2, $user->countSolutions());
     }
 
+    /** @test */
+    public function it_cannot_contain_htmltags_in_bio()
+    {
+        $user = factory(User::class)->create([
+            'bio' => '<strong>my bio</strong>',
+        ]);
+
+        $this->assertEquals('my bio', $user->bio());
+    }
+
```

Comment:
Move this to a separate `UpdateProfileTest` like the other job tests: https://github.com/laravelio/portal/tree/master/tests/Components/Jobs